main/website/index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang='en'>
+
+<head>
+    <meta charset='UTF-8'>
+    <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+    <link rel='icon' type='image/x-icon' href='data:image/png;base64,
+    AAABAAEAEBACAAAAAACwAAAAFgAAACgAAAAQAAAAIAAAAAEAAQAAAAAAQAAAAAAAAAAAAAAAAgAA
+    AAAAAAAAAAAAAPv/AAAAAAADwAAAD/AAABw4AAA73AAAN+wAAHfuAAB//gAAeZ4AAHmeAAA//AAA
+    P/wAAB/4AAAP8AAAA8AAAAAAAAD8PwAA8A8AAOAHAADAAwAAgAEAAIABAAAAAAAAAAAAAAAAAAAA
+    AAAAgAEAAIABAADAAwAA4AcAAPAPAAD8PwAA
+    '>
+    <title>Socketcand Adapter</title>
+    <script defer src='/alpine.js'></script>
+    <style>
+        body {
+            max-width: 1000px;
+            padding: 5pt;
+            margin: auto;
+            background-color: #ffe8ff;
+        }
+    </style>
+</head>
+
+<body>
+    <noscript>
+        <strong>
+            This site requires JavaScript. Please enable it.
+        </strong>
+    </noscript>
+    <h1>Socketcand Adapter</h1>
+    <p>This webpage is served by an ESP32-EVB Socketcand Adapter.</p>
+    <p>This adapter serves a minimal subset of
+        <a href='https://github.com/linux-can/socketcand/blob/master/doc/protocol.md'>Socketcand rawmode</a>.
+        on port 9999.
+    </p>
+
+    <h2>Status</h2>
+    <div x-data="{status: 'loading...', status_message: ''}" x-init="
+        console.log('initializing.');
+        try {
+            status = await (await fetch('/status')).json();
+            status_message = '';
+        } catch (error) {
+            status_message = `ERROR: Couldn't fetch status from server: ${error}`;
+        }
+        ">
+        <pre x-text="JSON.stringify(status, null, 2)"></pre>
```

Comment:
I think it would look a bit better if you looped through the settings with https://alpinejs.dev/directives/for

Reply:
Fixed in 72219c3.